src/assets/i18n/en.json5

```diff
@@ -5330,4 +5356,25 @@
   "admin.system-wide-alert.breadcrumbs": "System-wide Alerts",
 
   "admin.system-wide-alert.title": "System-wide Alerts",
+
+  "item-access-control-title": "This form allows you to perform changes to the access condition of all the item's metadata and all its bitstreams.",
+  "collection-access-control-title": "This form allows you to perform changes to the access condition of all the items owned by collection under this community. Changes can be performed on the access condition for the metadata (item) or for the content (bitstream).",
+  "community-access-control-title": "This form allows you to perform changes to the access condition of all the items owned by collection under this community. Changes can be performed on the access condition for the metadata (item) or for the content (bitstream).",
+  "access-control-item-header-toggle": "Item's Metadata",
+  "access-control-bitstream-header-toggle": "Bitstreams",
+  "access-control-mode": "Mode",
+  "access-control-access-conditions": "Access conditions",
+  "access-control-no-access-conditions-warning-message": "You have not specified any access conditions, the new items access conditions will be inherited from the owning collection.",
+  "access-control-replace-all": "Replace access conditions",
+  "access-control-add-to-existing": "Add to existing ones",
+  "access-control-limit-to-specific": "Limit the changes to specific bitstreams",
+  "access-control-process-all-bitstreams": "process all the bitstreams in the item",
+  "access-control-bitstreams-selected": "bitstreams selected",
+  "access-control-reset": "Reset",
+  "access-control-execute": "Execute",
+  "access-control-add-more": "Add more",
+  "access-control-select-bitstreams-modal.title": "Select bitstreams",
+  "access-control-select-bitstreams-modal.no-items": "No items to show.",
+  "access-control-select-bitstreams-modal.close": "Close",
```

Comment:
Please keep the empty lines between i18n comments. 

The reason for them is that if you run the sync-i18n script to add the new keys to the other languages, they'll get comments indicating which ones are new and which are the english equivalents

Without the blank lines in between them, it becomes hard to tell which comment corresponds to which message